publish-subscribe/publisher.bal

```diff
@@ -0,0 +1,28 @@
+import ballerina/http;
+import ballerinax/kafka;
+import ballerina/random;
+
+type MatchEvent record {|
+    string matchId;
+    string time;
+    string event;
+    string description;
+|};
+
+service /api on new http:Listener(8080) {
+
+    private final kafka:Producer kafkaPublisher;
+
+    function init() returns error? {
+        self.kafkaPublisher = check new(kafka:DEFAULT_URL);
+    }
+    
+    resource function post predict(MatchEvent event) returns error? {
+        check self.kafkaPublisher->send({
+            topic: event.matchId,
+            value:  calculateWinningOdds(event)
```

Comment:
I think we don't need `calculateWinningOdds`, can we just publish the whole even here?
We can think of the client as a ui for just displaying mach info.

Reply:
We can do this.